src/clj/game/cards/hardware.clj

```diff
@@ -437,6 +437,26 @@
    "HQ Interface"
    {:in-play [:hq-access 1]}
 
+   "Knobkierie"
+   {:implementation "MU usage restriction not enforced"
+    :in-play [:memory 3]
+    :events {:successful-run
+             {:delayed-completion true
```

Comment:
This tag is ignored when it's the same map as an `:optional`, `:psi`, or `:trace`.